### PR TITLE
Devengage 1290 self reference route

### PR DESCRIPTION
**Pull request by carnellj**

### CODE REVIEW

**Comment by ronanwatkins** [comment]
documentation is out of sync

File: genesyscloud/resource_genesyscloud_routing_email_route.go [hunk 1]
```diff
@@ -331,7 +362,19 @@ func readRoutingEmailRoute(ctx context.Context, d *schema.ResourceData, meta int
 		}
 
 		if route.ReplyEmailAddress != nil {
-			d.Set("reply_email_address", []interface{}{flattenQueueEmailAddress(*route.ReplyEmailAddress)})
+			log.Printf("I am in the route.ReplyEmailAddress != nil ")
```
Comment (ronanwatkins) [inline genesyscloud/resource_genesyscloud_routing_email_route.go:365]: No need for this log

**Comment by ronanwatkins** [comment]
You'll need to remove the [terraform-provider-genesyscloud](https://github.com/MyPureCloud/terraform-provider-genesyscloud/pull/311/files#diff-5e0086578b2d0f330d61c37fcd3f2fb83e33b76391e4619630af418392957a3f) binary that you added.

File: genesyscloud/resource_genesyscloud_routing_email_route.go (hunk 1 above)
Comment (carnellj) [reply genesyscloud/resource_genesyscloud_routing_email_route.go:365]: Done

**Pull request merged**